### PR TITLE
chore: update Material and MUX

### PR DESCRIPTION
**Pull request by Arieldelossantos**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Chore: update Material and MUX

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [ ] [Docs](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc) have been added/updated
- [x] [Runtime Tests and/or UI Tests](https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal)
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

**Review by agneszitte — CHANGES_REQUESTED** [review]
@Arieldelossantos please keep tabs instead of spaces please

File: src/Directory.Packages.props [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
+    <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.1.5" />
+    <PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
+    <PackageVersion Include="SkiaSharp.Views.Uno" Version="2.88.4-preview.84" />
+    <PackageVersion Include="SkiaSharp.Views.WinUI" Version="2.88.4-preview.84" />
+    <PackageVersion Include="SkiaSharp.Views.Uno.WinUI" Version="2.88.4-preview.84" />
+    <PackageVersion Include="Uno.Core.Extensions.Collections" Version="4.0.1" />
+    <PackageVersion Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
+    <PackageVersion Include="Uno.Core.Extensions.Logging" Version="4.0.1" />
+    <PackageVersion Include="Uno.Cupertino" Version="3.0.0-dev.332" />
+    <PackageVersion Include="Uno.Cupertino.WinUI" Version="3.0.0-dev.332" />
```
Comment (agneszitte) [inline src/Directory.Packages.props:17]: Please keep Uno.Cupertino in sync with Uno.Material also

File: samples/Directory.Packages.props [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+    <PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
+    <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.1.5" />
+    <PackageVersion Include="SkiaSharp.Skottie" Version="2.88.4-preview.84" />
+    <PackageVersion Include="SkiaSharp.Views" Version="2.88.4-preview.84" />
+    <PackageVersion Include="SkiaSharp.Views.Uno" Version="2.88.4-preview.84" />
+    <PackageVersion Include="SkiaSharp.Views.Uno.WinUI" Version="2.88.4-preview.84" />
+    <PackageVersion Include="Uno.Core.Extensions.Compatibility" Version="4.0.1" />
+    <PackageVersion Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
+    <PackageVersion Include="Uno.Cupertino" Version="3.0.0-dev.332" />
+    <PackageVersion Include="Uno.Cupertino.WinUI" Version="3.0.0-dev.332" />
```
Comment (agneszitte) [inline samples/Directory.Packages.props:28]: Please keep Uno.Cupertino in sync with Uno.Material also

**Review by agneszitte — CHANGES_REQUESTED** [review]

File: src/Directory.Packages.props (hunk 1 above)
Comment (Arieldelossantos) [reply src/Directory.Packages.props]: ready

File: samples/Directory.Packages.props (hunk 2 above)
Comment (Arieldelossantos) [reply samples/Directory.Packages.props]: ready

**Pull request merged**